src/functions.js

```diff
@@ -87,15 +87,12 @@ async function createFunction(memphisEvent, eventHandler, asJson = false) {
                         headers: processedHeaders,
                         payload: Buffer.from(processedMessage).toString('base64')
                     });
-                } else if (processedMessage === null && processedHeaders === null) {
```

Comment:
Why did you remove this part? I think it's still needed and only need to remove the next 'else if'

Reply:
You are correct. I don't know why I removed that one. I didn't do it to the python SDK and I did similar changes. I added it back.